<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dyuthi</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>DETATAILS OF THE OWNER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>VEHICLE NUMBER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>VEHICLE TYPE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_car">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>96</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CAR</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkbox_van">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>96</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>VAN</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkbox_bus">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>96</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>BUS</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkbox_jeep">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>96</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>JEEP</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkbox_lorry">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>96</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>LORRY</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LICENCE EXPIRE DATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INSURANCE EXPIRE DATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>EMISSION TEST EXPIRE DATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vehicleNumber">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_licenceExpire">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_insuranceExpire">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_emmisionExpire">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>FIRST NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LINE 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LINE 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LINE 3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addr1">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addr2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addr3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ADDRESS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NIC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BLOOD GROUP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firstname">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LAST NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lastname">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_licneceRegNum">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_blood">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_validvehiclenumber">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dyuthi</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>DETATAILS OF THE VEHICLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>480</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ACC NO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BANK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AccNo">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>480</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="bankList">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>430</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SAMPATH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COMERCIAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DFCC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PEOPEL'S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SEYLAN</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>OWNER LICENCE EXPIRE DATE</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_ownerLicneceExpire">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>eg: KP-9375</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>checkbox_car</zorder>
  <zorder>checkbox_van</zorder>
  <zorder>checkbox_bus</zorder>
  <zorder>checkbox_jeep</zorder>
  <zorder>checkbox_lorry</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>lineEdit_vehicleNumber</zorder>
  <zorder>dateEdit_licenceExpire</zorder>
  <zorder>dateEdit_insuranceExpire</zorder>
  <zorder>dateEdit_emmisionExpire</zorder>
  <zorder>pushButton_submit</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>lineEdit_addr1</zorder>
  <zorder>lineEdit_addr2</zorder>
  <zorder>lineEdit_addr3</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>lineEdit_firstname</zorder>
  <zorder>label_16</zorder>
  <zorder>lineEdit_lastname</zorder>
  <zorder>lineEdit_licneceRegNum</zorder>
  <zorder>lineEdit_blood</zorder>
  <zorder>label</zorder>
  <zorder>label_validvehiclenumber</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>lineEdit_AccNo</zorder>
  <zorder>bankList</zorder>
  <zorder>label_20</zorder>
  <zorder>dateEdit_ownerLicneceExpire</zorder>
  <zorder>pushButton_clear</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
